adapters/33across/33across.go

```diff
@@ -114,6 +114,7 @@ func (a *TtxAdapter) makeRequest(request openrtb2.BidRequest, impList []openrtb2
 		Uri:     a.endpoint,
 		Body:    reqJSON,
 		Headers: headers,
+		ImpIDs:  openrtb_ext.GetImpIDs(impList),
```

Comment:
Consider replacing `impList` with `request.Imp` or `request` in this call. `GetImpIDsFromRequest(request)` would look nice. There are many adapters were passing the request would be simpler.

Reply:
Thanks @SyntaxNode.
Replaced with `request.Imp`.
But if we use `GetImpIDsFromRequest(request)`, Do we need to consider whether it's necessary to pass the entire request data to the function, given that only the `imp[].id` contained in `request.Imp` is needed.